help/quote-or-forward-a-message.md

```diff
@@ -1,19 +1,24 @@
 # Quote or forward a message
 
-You can quote a previous message, either replying within a topic or to
-start a new topic.
+Conversations in Zulip are [organized by topic](/help/introduction-to-topics),
+so it's generally clear what you're responding to when you send a message.
+However, when responding to an older message, you may want to
```

Comment:
I think quoting a part of a recent message is quite useful too, and should be called out here -- quoting just the one sentence that you want to reply to in a giant message is a good habit to recommend.